Library/Homebrew/test/cask/cmd/info_spec.rb

```diff
@@ -141,6 +142,24 @@
 
   it "can run be run with a url twice", :needs_network do
     expect {
+      Utils::Analytics.formulae_api_json
+    .should_receive(:endpoint)
+    .with("test")
+    .and_return("{
+      analytics: {
+        install: {
+          30d: {
+            docker: 1000
+          },
+          90d: {
+            docker: 2000
+          },
+          365d: {
+            docker: 3000
+          }
+        }
+      }
+    }")
```

Comment:
You need something like this (untested so you'll need to test locally):

```suggestion
  it "can run be run with a url twice", :needs_network do
    expect(Utils::Analytics).to receive(:formulae_api_json).with("docker").and_return("{
      analytics: {
        install: {
          30d: {
            docker: 1000
          },
          90d: {
            docker: 2000
          },
          365d: {
            docker: 3000
          }
        }
      }
    }")
    expect {
```